Composer/packages/client/src/recoilModel/dispatchers/provisionQNA.ts

```diff
@@ -6,7 +6,7 @@ import { CallbackInterface, useRecoilCallback } from 'recoil';
 import { CognitiveServicesManagementClient } from '@azure/arm-cognitiveservices';
 import { TokenCredentials } from '@azure/ms-rest-js';
 import { SearchManagementClient } from '@azure/arm-search';
-import { SkuName } from '@azure/arm-search/lib/models';
+import { SkuName } from '@azure/arm-search/esm/models';
```

Comment:
just curious: why was this changed?

Reply:
Yarn 3 resolved packages a bit differently and in the never package types moved to esm folder